packages/botonic-plugin-ai-agents/src/constants.ts

```diff
@@ -1,2 +1,8 @@
 export const HUBTYPE_API_URL =
   process.env.HUBTYPE_API_URL || 'https://api.hubtype.com'
+
+export const AZURE_OPENAI_API_KEY = process.env.AZURE_OPENAI_API_KEY // pragma: allowlist secret
+export const AZURE_OPENAI_API_DEPLOYMENT_NAME =
+  process.env.AZURE_OPENAI_API_DEPLOYMENT_NAME
+export const AZURE_OPENAI_API_VERSION = process.env.AZURE_OPENAI_API_VERSION
+export const AZURE_OPENAI_API_BASE = process.env.AZURE_OPENAI_API_BASE
```

Comment:
We could define some default values for **some** this env variables in case they are not passed:

```
export const AZURE_OPENAI_API_DEPLOYMENT_NAME = process.env.AZURE_OPENAI_API_DEPLOYMENT_NAME || 'gpt-4o-mini_p1'
export const AZURE_OPENAI_API_VERSION = process.env.AZURE_OPENAI_API_VERSION || '2024-12-01-preview'
```